core-bundle/src/Resources/contao/languages/de/default.xlf

```diff
@@ -1957,6 +1957,14 @@
         <source>descending</source>
         <target>absteigend</target>
       </trans-unit>
+      <trans-unit id="MSC.disposition.attachment">
+        <source>Dispose as download</source>
+        <target>Als Download ausspielen</target>
+      </trans-unit>
+      <trans-unit id="MSC.disposition.inline">
+        <source>Dispose in browser</source>
```

Comment:
I don't like`dispose` here. To me it sounds like throwing something away. I'd prefer something like `Provide as attachment (download)` and `Provide inline (within Browser)`.